.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+
+env:
+  - COMPOSER_FLAGS="--prefer-stable"
+
+php:
+  - 7.1
+
+matrix:
+  include:
+    - php: 7.1
+    - script: vendor/bin/phpcs -n
+      env: PHPCS_BUILD=1
+    - php: 7.1
+  allow_failures:
+    - php: 7.1
```

Comment:
please remove the allowed_failures. We do not wat to exempt a version. This is usually something you do for a nightly or beta stuff. This should work.